radicale_imap/__init__.py

```diff
@@ -32,16 +32,44 @@ class Auth(BaseAuth):
     type = radicale_imap
     imap_host = imap.server.tld
     imap_secure = True
+    imaps = True # for "old"-style imap over tls
```

Comment:
Change ``True`` to ``False``. I think ``IMAP over SSL`` is better.

Reply:
I probably named this badly.
What I mean with "old-style" is that it uses a dedicated port for imap over SSL/TLS, which I consider to be more secure than upgrading with STARTTLS.
The proper imap over SSL/TLS is not even supported without this patch.